CONTRIBUTING.md

```diff
@@ -134,9 +134,20 @@ Prettier before a commit by default.
 When adding or changing a service [please write tests][service-tests], and ensure the [title of your Pull Requests follows the required conventions](#running-service-tests-in-pull-requests) to ensure your tests are executed.
 When changing other code, please add unit tests.
 
-To run the integration tests, you must have Redis installed and in your PATH.
-Use `brew install redis`, `yum install redis`, etc. The test runner will
-start the server automatically.
+The integration tests are not run by default. For most contributions it is OK to skip these unless you're working directly on the code for storing the GitHub token pool in posgres/redis.
+
+To run the integration tests:
+
+- You must have Redis installed and in your PATH. Use `brew install redis`, `apt-get install redis`, etc. The test runner will start the server automatically.
```

Comment:
Do you think it would be worth going ahead and nuking this? Understand it's accurate atm, but I think that's likely to change fairly quickly (guessing your plan is to delete this when we drop the redis portion of the codebase?)

Reply:
I think it can be removed when I do https://github.com/badges/shields/issues/8932

For the moment, if you try to run the integration tests without redis they won't work so I think it is worth keeping it in the docs for the moment